Makefile

```diff
@@ -321,3 +318,59 @@ include Makefile.test
 include src/main/scala/device/standalone/standalone_device.mk
 
 .PHONY: verilog sim-verilog emu clean help init bump bsp $(REF_SO)
+
+#
+# Docker Support
+#
+DOCKER_TARGETS := $(TOP_V) $(SIM_TOP_V)
+MILL_WORK_DIR := /work
+MILL_OUTPUT_DIR := ./out_docker
```

Comment:
maybe we should use a `hidden` directory for docker mill out (e.g. `.docker-mill-out`?), and add it to `.gitignore`?

Reply:
fixed, use .docker-mill-out for docker mill intermedia files